src/burnchains/mod.rs

```diff
@@ -1053,6 +1053,15 @@ pub mod test {
             txop.txid =
                 Txid::from_test_data(txop.block_height, txop.vtxindex, &txop.burn_header_hash, 0);
 
+            let epoch = SortitionDB::get_stacks_epoch(ic, txop.block_height)
+                .unwrap()
+                .expect(&format!("BUG: no epoch for height {}", &txop.block_height));
+            if epoch.epoch_id == StacksEpochId::Epoch2_05
```

Comment:
You should be able to use `epoch.epoch_id >= StacksEpochId::Epoch2_05`.  Also, we'll have a different epoch marker for 2.1, so please open an issue to add it and fix this test.